tests/test_api.py

```diff
@@ -0,0 +1,100 @@
+from typing import Literal
+
+import pytest
+from pytest_mock import MockerFixture
+
+from corva.application import Corva
+from corva.configuration import SETTINGS
+
+
+@pytest.fixture(scope='function')
+def event():
+    return (
+        '[{"records": [{"asset_id": 0, "company_id": 0, "version": 0, "collection": "", "data": {}}], '
+        '"metadata": {"app_stream_id": 0, "apps": {"%s": {"app_connection_id": 0, "app_version": 0}}}}]'
+    ) % SETTINGS.APP_KEY
+
+
+def app(event, api, cache):
+    return api
+
+
+def test_default_headers(event, mocker: MockerFixture):
+    request_patch = mocker.patch('requests.request')
+
+    api = Corva(context=None).stream(app, event)[0]
+
+    api.get('')  # do some api call
+
+    request_patch.assert_called_once()
+    assert 'Authorization' in request_patch.call_args.kwargs['headers']
+    assert 'X-Corva-App' in request_patch.call_args.kwargs['headers']
+
+
+@pytest.mark.parametrize(
```

Comment:
This is a typo here right?

Reply:
ci (tests) would fail if there was a typo

Reply:
LOL how did I never notice this misspelling in pytest? Ridiculous.